types/taiko/index.d.ts

```diff
@@ -0,0 +1,366 @@
+    // https://taiko.gauge.org/#rightclick
+    export function rightClick(selector: SearchElement | MouseCoordinates, options?: BasicNavigationOptions, args?: string[]): Promise<void>;
+    // https://taiko.gauge.org/#draganddrop
+    export function dragAndDrop(
+        source: SearchElement,
+        destination: SearchElement,
+        distance: DragAndDropDistance
+    ): Promise<void>;
+    // https://taiko.gauge.org/#hover
+    export function hover(selector: SearchElement, options?: EventOptions): Promise<void>;
+    // https://taiko.gauge.org/#focus
+    export function focus(selector: SearchElement, options?: EventOptions): Promise<void>;
+    // https://taiko.gauge.org/#write
+    export function write(text: string, into?: SearchElement, options?: WriteOptions): Promise<void>;
+    // https://taiko.gauge.org/#clear
+    export function clear(selector: SearchElement, options?: NavigationOptions): Promise<void>;
+    // https://taiko.gauge.org/#attach
+    export function attach(filepath: string, to: SearchElement): Promise<void>;
+    // https://taiko.gauge.org/#press
+    export function press(keys: string | string[], options?: KeyOptions): Promise<void>;
+    // https://taiko.gauge.org/#highlight
+    export function highlight(selector: SearchElement): Promise<void>;
+    // https://taiko.gauge.org/#mouseaction
+    export function mouseAction(action: 'press' | 'move' | 'release', coordinates: MouseCoordinates, options?: NavigationOptions): Promise<void>;
+    // https://taiko.gauge.org/#scrollto
+    export function scrollTo(selector: SearchElement, options?: EventOptions): Promise<void>;
+    // https://taiko.gauge.org/#scrollright
+    export function scrollRight(selector?: SearchElement | number, px?: number): Promise<void>;
+    // https://taiko.gauge.org/#scrollleft
+    export function scrollLeft(selector?: SearchElement | number, px?: number): Promise<void>;
+    // https://taiko.gauge.org/#scrollup
+    export function scrollUp(selector?: SearchElement | number, px?: number): Promise<void>;
+    // https://taiko.gauge.org/#scrolldown
+    export function scrollDown(selector?: SearchElement | number, px?: number): Promise<void>;
+    // https://taiko.gauge.org/#screenshot
+    export function screenshot(options?: ScreenshotOptions, ...args: SearchElement[]): Promise<Buffer>;
+    // https://taiko.gauge.org/#tap
+    export function tap(selector: SearchElement, options?: TapOptions, ...args: SearchElement[]): Promise<void>;
+
+    /**
+     * Selectors
+     */
+
+    // https://taiko.gauge.org/#dollar
+    export function $(selector: string, ...args: SearchElement[]): Selector;
+    // https://taiko.gauge.org/#image
+    export function image(selector: SearchElement, ...args: SearchElement[]): SearchElement;
+    // https://taiko.gauge.org/#link
+    export function link(selector: SearchElement, ...args: SearchElement[]): SearchElement;
```

Comment:
can take option that says {selectHiddenElement:true|false}

Reply:
added